android/app/src/main/java/com/tourneynizer/tourneynizer/requesters/UserRequester.java

```diff
@@ -0,0 +1,172 @@
+package com.tourneynizer.tourneynizer.requesters;
+
+import android.content.Context;
+import android.util.Log;
+
+import com.android.volley.AuthFailureError;
+import com.android.volley.Request;
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.JsonArrayRequest;
+import com.android.volley.toolbox.JsonObjectRequest;
+import com.android.volley.toolbox.StringRequest;
+import com.tourneynizer.tourneynizer.model.User;
+import com.tourneynizer.tourneynizer.util.CookieRequestFactory;
+import com.tourneynizer.tourneynizer.util.HTTPRequester;
+import com.tourneynizer.tourneynizer.util.JSONConverter;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.sql.Time;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by ryanwiener on 2/16/18.
+ */
+
+public class UserRequester {
+
+    public interface OnUserLoadedListener {
+        public void onUserLoaded(User user);
+    }
+
+    public static void getUserFromEmail(Context c, String email, final OnUserLoadedListener listener) {
```

Comment:
Why not make this class a singleton with a single register context service to avoid passing the context in every time?